addons/survey/views/survey_survey_views.xml

```diff
@@ -191,74 +191,163 @@
+                                bg_color="bg-danger"
+                                attrs="{'invisible': [('active', '=', True)]}"/>
                         </div>
-                        <div class="o_kanban_record_top">
-                                <h4 class="o_kanban_record_title p-0 mb4"><field name="title" /></h4>
-                        </div>
-                        <div class="row">
-                            <div class="col-10 p-0 pb-1">
-                                <div class="container o_kanban_card_content" t-if="record.answer_done_count.raw_value != 0">
-                                    <div class="row mt-4 ml-5">
-                                        <div class="col-4 p-0">
-                                            <a name="action_result_survey" type="object" class="d-flex flex-column align-items-center">
-                                                <span class="font-weight-bold"><field name="answer_done_count"/></span>
-                                                <span class="text-muted">Answers</span>
-                                            </a>
-                                        </div>
-                                        <div class="col-4 p-0 border-left" t-if="record.scoring_type.raw_value != 'no_scoring'" >
-                                            <a name="action_survey_user_input_certified" type="object" class="d-flex flex-column align-items-center">
-                                                <span class="font-weight-bold"><field name="success_count"/></span>
-                                                <span class="text-muted" t-if="!record.certification.raw_value">Passed</span>
-                                                <span class="text-muted" t-else="">Certified</span>
-                                            </a>
-                                        </div>
-                                        <div class="col-4 p-0 border-left" t-if="record.scoring_type.raw_value != 'no_scoring'" >
-                                            <a name="action_survey_user_input_completed" type="object" class="d-flex flex-column align-items-center">
-                                                <span class="font-weight-bold"> <t t-esc="Math.round(record.success_ratio.raw_value)"></t> %</span>
-                                                <span class="text-muted" >Success</span>
-                                            </a> 
-                                        </div>
-                                    </div>
+                        <!-- displayed in grouped mode -->
+                        <div class="o_survey_kanban_card_grouped">
+                            <widget name="web_ribbon" title="Archived"
+                                bg_color="bg-danger"
+                                attrs="{'invisible': [('active', '=', True)]}"/>
+                            <div class="o_dropdown_kanban dropdown" t-if="widget.editable">
```

Comment:
Since list-mode is the default, do we need all this burger menu anymore?
It will still be usable but you have to group first, seems overkill to keep the code.

(Maybe for the color thing?)

Reply:
Boarf, it was present before, still present after. Will not make another round of testing just for that.